### PR TITLE
Ages by Decade

### PR DESCRIPTION
**Pull request by KaiBot3000**
- [x] arrange ages by decade i.e. "20's"
- [x] separate minors into their own category
- [x] add me to `CONTRIBUTORS` :P

refs https://github.com/drnikki/open-demographics/issues/5

review and feedback very welcome!

### CODE REVIEW

File: age.md [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
     () Prefer not to answer
 
 ### Short Question
-    What is your age range?
-    () 0-24
-    () 25-44
-    () 45-64
-    () 65+
+    () 0-19
+    () 20-30
```
Comment (drnikko) [inline age.md]: I think you might have meant 20-39.  Folks in their 30s need a way to participate. :)

**Comment by drnikko** [comment]
also, please merge master into your branch when you get a chance.  I merged another PR and there's conflict on the CONTRIBUTORS file.   Thank you for contributing!!

File: age.md (hunk 1 above)
Comment (drnikko) [reply age.md]: @KaiBot3000 ^

**Comment by KaiBot3000** [comment]
Thanks for the ping!
I:

- [x] merged in from `master` and resolved the conflict
- [x] fixed the typo excluding people in their 30's from the short question

I'm happy to squash commits if that's our style, too.

**Comment by drnikko** [comment]
Thank you!! I don't know that we have a _huge_ need to squash commits at this point.  No need to make more work for anyone.  <3

**Pull request merged**